### ISSUE TITLE
gosnappi appending values when call SetValues multiple times

### ISSUE THREAD

**Issue by alakendu** [open-traffic-generator/openapiart#179]
**Version information**
v0.1.9

**Describe the bug**
gosnappi appending values when call SetValues multiple times

**To Reproduce**
Try this code snippet 

```go
	api := openapiart.NewApi()
	config := api.NewPrefixConfig()
	config.SetListOfStringValues([]string{"111", "222"})
	config.SetListOfStringValues([]string{"333", "444"})
```
I am getting this 
```
[111 222 333 444]
````

**Expected behavior**
I am expecting it should overwrite and this should be the value
```
[333 444]
```

**Additional context**
We need to revisit setDefault if we are ok with this approach (append new value)

**Comment by ajbalogh** [comment]
The expected behavior should be to overwrite and setDefault should follow that.

**Comment by alakendu** [comment]
This address in https://github.com/open-traffic-generator/openapiart/pull/177

**Comment by alakendu** [comment]
fix should available 0.1.11
